app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -700,8 +702,12 @@ void OpenXRInputSource::EmulateControllerFromHand(device::RenderMode renderMode,
     delegate.SetCapabilityFlags(mIndex, flags);
 
     // Squeeze action
-    bool middlePinching = GetDistanceBetweenJoints(XR_HAND_JOINT_THUMB_TIP_EXT,
+    bool middlePinching = false;
+    if ((mHandJoints[XR_HAND_JOINT_THUMB_TIP_EXT].locationFlags & XR_SPACE_LOCATION_POSITION_VALID_BIT) != 0 &&
```

Comment:
Mind creating a private method that does this check? This will make this if and the above one more understandable.

Reply:
Agree, will fix that.

Reply:
Fixed